src/app/backend/client/manager.go

```diff
@@ -20,7 +20,6 @@ import (
 	"log"
 	"strings"
 
-	"github.com/emicklei/go-restful"
```

Comment:
Install and use [this](https://godoc.org/golang.org/x/tools/cmd/goimports) `go get golang.org/x/tools/cmd/goimports`. I have used it and it doesn't delete anything.